doc/changes/devel.rst

```diff
@@ -29,7 +29,7 @@ Enhancements
 
 Bugs
 ~~~~
-- None yet
+- Fix bugs with saving splits for `Epochs` (:gh:`11876` by `Dmitrii Altukhov`_) 
```

Comment:
I'm a bit surprised this worked, should be 

```suggestion
- Fix bugs with saving splits for :class:`~mne.Epochs` (:gh:`11876` by `Dmitrii Altukhov`_) 
```

Reply:
I'll wait for CIs to finish then manually merge after commiting this tiny change

Reply:
our default `currentmodule` is `mne` and our `default_role = "py:obj"` in `conf.py`, hence it auto-resolved to `:class:`mne.Epochs`. But I agree we should change it as we've gone to prefering explicit links